stdlib/public/core/BidirectionalCollection.swift

```diff
@@ -276,7 +278,9 @@ extension BidirectionalCollection {
   ///   collection. `n` must be greater than or equal to zero.
   /// - Returns: A subsequence that leaves off `n` elements from the end.
   ///
-  /// - Complexity: O(*n*), where *n* is the number of elements to drop.
+  /// - Complexity: O(1) if the collection conforms to
+  ///   `RandomAccessCollection`; otherwise, O(*k*), where *k* is the number of
+  ///   elements to drop.
   @inlinable // protocol-only
   public func dropLast(_ n: Int) -> SubSequence {
```

Comment:
I don't think we should change the doc in this way without also changing the parameter name to k.